### PR TITLE
Reviewed changes to path tracing documentation

### PR DESCRIPTION
**Pull request by Vic-Cooper**
As per the changes made in [#5551](https://github.com/Unity-Technologies/Graphics/pull/5551)

### CODE REVIEW

**Comment by Vic-Cooper** [comment]
@eturquin I changed your initial layout and notes so please check if I preserved your meaning.

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 1]
```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
...
-In path tracing, the *Transmission* option of subsurface scattering will only take effect if the surface is also set to be *Double-Sided* (any normal mode will do), in which case it will receive light from both sides.
+When you use path tracing, the **Transmission** option of subsurface scattering only takes effect if the surface Material is also set to be **Double-Sided**.  This means the surface can receive light from both sides.
 
-Here is an example of a sheet of fabric, lit from below by a point light:
+The following example images display the difference between of a sheet of fabric lit from below by a point light with a single-sided surface and double-sided surface:
 
 ![](Images/Path-traced-SSS-Single-sided.png)
 
-Single-sided or no Transmission
+Single-sided with no Transmission.
```
Comment (eturquin) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:86]: Should be "Single-sided, or no Transmission"

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 2]
```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
...
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Box**          | A box-shaped model where incident light enters through a flat surface and leaves through a flat surface.<br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Sphere**       | A sphere-shaped model that produces a magnifying glass-like effect to refraction. <br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Thin**         | A thin box surface type, similar to **Box** with a fixed thickness of 5cm. Select this for thin window-like surfaces. <br/>The behaviour of the **Thin** refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a window. | A double-sided Material that has its **Normal mode** set to  **Flip** or **Mirror**. |
```
Comment (eturquin) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:69]: | **Thin** | A thin surface type, characterized by infinitesimal thickness. Select this for thin window-like surfaces. <br/>The behaviour of the **Thin** refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a glass window. | A double-sided Material that has its **Normal mode** set to **Flip** or **Mirror**. |

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 3]
```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
...
-| *Thin*            | *Thin* object (e.g soap bubble or window)         | Double sided, with *Flip* or *Mirror* normal mode |
-| *None*            | *Thin* object, with smoothness = 1 and no Fresnel | Double sided, with *Flip* or *Mirror* normal mode |
+| Refraction model | Path tracing behavior                                        | Compatible Surface sides                                     |
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| **Box**          | A box-shaped model where incident light enters through a flat surface and leaves through a flat surface.<br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
+| **Sphere**       | A sphere-shaped model that produces a magnifying glass-like effect to refraction. <br/><br/>Suitable for thick objects like a paperweight or a crystal ball. | A double-sided Material that has its **Normal mode** set to **None**. |
```
Comment (eturquin) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:68]: Box and Sphere are **exactly** the same in path tracing, they correspond to a **thick** mode. It's a bit confusing to have them separate in this table, as if they have different effects.

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 4]
```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
...
+| **Thin**         | A thin box surface type, similar to **Box** with a fixed thickness of 5cm. Select this for thin window-like surfaces. <br/>The behaviour of the **Thin** refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a window. | A double-sided Material that has its **Normal mode** set to  **Flip** or **Mirror**. |
+| **None**         | A thin refractive surface, hardcoded to be fully smooth to simulate alpha blending.<br/>The behaviour of the **None **refraction model does not change with path tracing enabled .<br/><br/>Suitable for thin objects like a soap bubble or a window. | A double-sided Material that has its **Normal mode** set to  **Flip** or **Mirror**. |
```
Comment (eturquin) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:70]: I wouldn't say that this mode is suitable for anything really, it's just here to keep compatibility with raster mode when you switch to path tracing, but other than that **thin** should be prefered in every single situation.

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 5]
```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
 ## Materials parameterization
 
-Some phenomena like refraction, absorption in transmissive objects, or subsurface scattering, can be expressed more naturally in a path tracing setting than in its rasterized counterpart, and as such require less material parameters (e.g. additional thickness information, expected shape of the refractive object, ...). For that reason, some parameters have no effect in path tracing, while others bear a slightly different meaning.
+HDRP can perform effects like light refraction and absorption in transmissive objects, or subsurface scattering, more easily when you use path tracing setting than in rasterization . This means they need fewer Material parameters (e.g. additional thickness information, expected shape of the refractive object, ...). For that reason, some parameters have no effect in path tracing, and some parameters change behaviour. For more information, see [Refraction models](#).
```
Comment (eturquin) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:57]: It's not performing those effects "more easily" (as in, it's not computing them faster, or with less resources), but rather can express them "more naturally" given the different way of representing light transport simulation.

**Review by eturquin — CHANGES_REQUESTED** [review]

**Comment by Vic-Cooper** [comment]
Thanks for your feedback @eturquin I'm implementing your suggested changes in a[ google doc](https://docs.google.com/document/d/1dC2syNIT1hGV-Bz64RGClzdZfXbUVhx0sSiKZ9eQYUg/edit) so I can get it reviewed by docs editorial.

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md (hunk 5 above)
Comment (eturquin) [reply com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:57]: > Thanks for your feedback @eturquin I'm implementing your suggested changes in a[ google doc](https://docs.google.com/document/d/1dC2syNIT1hGV-Bz64RGClzdZfXbUVhx0sSiKZ9eQYUg/edit) so I can get it reviewed by docs editorial.

Sounds good! :)

**Comment by eturquin** [comment]
> Thanks for your feedback @eturquin I'm implementing your suggested changes in a[ google doc](https://docs.google.com/document/d/1dC2syNIT1hGV-Bz64RGClzdZfXbUVhx0sSiKZ9eQYUg/edit) so I can get it reviewed by docs editorial.

Sounds good! :)

**Pull request closed without merging**